tests/unit/test_config.py

```diff
@@ -53,6 +54,19 @@ def test_update_status_config(self):
         actual = update_status_config(milestones_json)
         self.assertEqual(actual, expected)
 
+    def test_get_variation_from_env(self):
+        """Make sure it doesn't blow up when it shouldn't, and does when
+        it should.
+        """
```

Comment:
nit: Probably a different docstrings ;) I know my docstrings are boring. :p 

```suggestion
"""
Check the conversion from string to tuples of two int.
"""
```